룸에서 맵이 보이던 문제 fix

orchis-seok opened the pull request:
#74

orchis-seok committed:
룸에서 맵이 보이던 문제 fix

glwlsl0716 reviewed (commented):
여전히 재현됩니다.

orchis-seok committed:
방장이 아닌 클라이언트 방 재입장시 맵이 보이던 문제 fix

glwlsl0716 reviewed (approved):
activeScene name 말고 buildIndex로 하면 더 좋을 거 같아여

orchis-seok committed:
피드백 반영

SceneManager.GetActiveScene().name.Equals("GamePlay Test")
->
SceneManager.GetActiveScene().buildIndex.Equals(1)
